winter/pagination/page_position_argument_inspector.py

```diff
@@ -39,12 +39,16 @@ def inspect_parameters(self, route: 'Route') -> List[openapi.Parameter]:
         parameters.append(self.limit_parameter)
         parameters.append(self.offset_parameter)
 
-        allowed_order_by_fields = get_allowed_order_by_fields(route.method)
-        if allowed_order_by_fields:
-            allowed_order_by_fields = ','.join(map(str, allowed_order_by_fields))
+        order_by_annotation = route.method.annotations.get_one_or_none(OrderByAnnotation)
+        if order_by_annotation:
+            allowed_order_by_fields = ','.join(map(str, order_by_annotation.allowed_fields))
+            default_part = f' Default is "{order_by_annotation.default}"' if order_by_annotation.default else ''
             order_by_parameter = openapi.Parameter(
                 name=self._page_position_argument_resolver.order_by_name,
-                description=f'Comma separated order by fields. Allowed fields: {allowed_order_by_fields}',
+                description=(
+                    f'Comma separated order by fields. '
+                    f'Allowed fields: {allowed_order_by_fields}.{default_part}'
```

Comment:
Why default_part is in the description and not in openapi.Parameter default property?